backend/btrixcloud/colls.py

```diff
@@ -764,11 +764,13 @@ async def list_urls_in_collection(
         crawl_ids = await self.get_collection_crawl_ids(coll_id)
 
         match_query: dict[str, object] = {"oid": oid, "crawl_id": {"$in": crawl_ids}}
+        sort_query: dict[str, int] = {"count": -1, "_id": 1}
```

Comment:
@tw4l I did a quick pass on this to get it working, feel free to push any suggestions directly!